mkosi/__init__.py

```diff
@@ -702,6 +704,15 @@ def prepare_grub_bios(state: MkosiState, partitions: Sequence[Partition]) -> Non
 
     initrd = Path(shutil.copy2(initrd, dst / "initrd"))
 
+    grubsnippet = textwrap.dedent(
+        """\
+        menuentry "{distribution}-{kver}" {{
+        linux /{kimg} {root} {cmdline}
+        initrd /{initrd} /{kmods}
+        }}
+        """
+    )
```

Comment:
I'd prefer to avoid using the formatting stuff, can't we just introduce some extra variables here instead and use those to satisfy the line limit?

Reply:
I'll have a look

Reply:
Changed it like that. In this case I prefer calling format explicitly, since it doesn't leak all the new variable names into the surrounding scope. I think it should be fine if tests pass, since the reused variable names should be the last point of use in each case.